.ci.yaml

```diff
@@ -17,7 +17,7 @@ platform_properties:
         [
           {"name":"builder_linux_framework","path":"builder"},
           {"name":"android_sdk","path":"android"},
-          {"name":"chrome_and_driver","path":"chrome"},
+          {"name":"chrome_and_driver","path":"chrome_96"},
```

Comment:
Actually we only need to change the name:

`{"name":"chrome_and_driver_96","path":"chrome"},`

The path is fixed and mounted at runtime from the named cache.

Reply:
Changing the name results in recipe failures, so I can't use that method.

Reply:
Can you please share the one with recipe failures to take a look?

Reply:
We are still missing to provide the version on all the chrome_and_driver dependencies.